src/app/components/post-detail/post-detail.component.html

```diff
@@ -1,25 +1,29 @@
 <div class="container post-container">
         <div class="card mb-4 post-card">
             <div class="card-header">
-            <h4>{{post.title}}</h4>
+            <h4>{{getPost().title}}</h4>
```

Comment:
why use the `getPost()` method? we have the `post` object to explicitly bind to.

Reply:
because post is private

Reply:
I've seen that error in WS too, I'm not sure why. We still have access to private members of the TS within the bound HTML template though.

Reply:
okie dokie, i will repeal that change then. what about the other change?